api/src/main/kotlin/edu/wgu/osmt/RoutePaths.kt

```diff
@@ -1,36 +1,51 @@
 package edu.wgu.osmt
 
+import org.apache.commons.lang3.StringUtils
+
 object RoutePaths {
+
+
+    const val LEGACY = "/v2"
+    const val LATEST = "/v3"
+    const val DEFAULT = LATEST
+    const val UNVERSIONED = StringUtils.EMPTY
+
     const val API = "/api"
-    const val SEARCH_PATH = "$API/search"
-    const val EXPORT = "$API/export"
+    const val VERSIONED_API = "/api/{apiVersion}"
+    const val UNVERSIONED_API = API
+    private const val SEARCH_PATH = "/search"
+
+    //export
+    private const val EXPORT = "/export"
     const val SEARCH_SKILLS = "$SEARCH_PATH/skills"
-    const val EXPORT_LIBRARY = "$EXPORT/library"
+    private const val EXPORT_LIBRARY = "$EXPORT/library"
     const val EXPORT_LIBRARY_CSV = "$EXPORT_LIBRARY/csv"
     const val EXPORT_LIBRARY_XLSX = "$EXPORT_LIBRARY/xlsx"
+    private const val EXPORT_SKILLS = "$EXPORT/skills"
+    const val EXPORT_SKILLS_CSV = "$EXPORT_SKILLS/csv"
+    const val EXPORT_SKILLS_XLSX = "$EXPORT_SKILLS/xlsx"
     const val SEARCH_SIMILAR_SKILLS = "$SEARCH_SKILLS/similarity"
     const val SEARCH_SIMILARITIES = "$SEARCH_SKILLS/similarities"
     const val SEARCH_COLLECTIONS = "$SEARCH_PATH/collections"
 
-    const val SKILLS_PATH = "$API/skills"
+    //skills
+    private const val SKILLS_PATH = "/skills"
     const val SKILLS_LIST = SKILLS_PATH
```

Comment:
This route is missing v3 api support in the controller

Reply:
yes for v3 the corresponding endpoint is the filtered one, since we implemented it after 2.5